app/js/components/FileViewer.vue

```diff
@@ -272,35 +272,49 @@ export default {
 		},
 		getTimestampFromLine(row) {
 			const line = this.viewer.session.getLine(row);
-			const regex = /^\s*(\d{4}\W\d{2}\W\d{2}(\s+|T)\d{2}:\d{2}:\d{2}(\.\d+)?)/;
+			const regex = /\s*(\d{4}\W\d{2}\W\d{2}(\s+|T)\d{2}:\d{2}:\d{2}(\.\d+)?)/;
 			const matches = line.match(regex);
+
 			if (matches && matches[1]) {
 				let timestamp = Date.parse(matches[1]);
 				return !isNaN(timestamp) ? timestamp : 0;
 			}
 			return 0;
 		},
-		scrollToTimestamp(timestamp, scrollOffset, cursorColumn) {
+		getTimestampFromParentLine(row) {
+
+			let timeStampInLine = 0;
+			let parentRow = row;
+			let line;
+			while (timeStampInLine === 0) {
+				line = this.viewer.session.getLine(parentRow);
+
+				timeStampInLine = this.getTimestampFromLine(parentRow,false);
+				parentRow--;
+				if (parentRow < 0)
+					return 0;
+			}
+			return timeStampInLine;
+		},
+		scrollToTimestamp(timestamp, cursorColumn) {
+
 			if (!this.syncTimestamps) {
 				return;
 			}
 
 			const row = this.findRowAtTimestamp(timestamp);
 			if (row !== false) {
-				// gotoLine() needs to be called before setScrollTop().
-				// Otherwise, the broken editor scrolling logic will mess up the scroll position.
+				this.viewer.scrollToLine(row);
+				this.viewer.gotoLine(row + 1, cursorColumn, false);
+			} else {
+				this.viewer.scrollToLine(row);
 				this.viewer.gotoLine(row + 1, cursorColumn, false);
-
-				// We need to scroll manually here because editor.scrollToLine() doesn't work correctly.
-				const scrollTop = this.getRowOffset(row) + scrollOffset;
-				this.viewer.session.setScrollTop(scrollTop);
 			}
 		},
 		getRowOffset(row) {
-			const pos = this.viewer.session.documentToScreenPosition({ row: row, column: 0 });
 			// FIXME Get line height from editor (or fix editor scrolling).
 			//  Only works with the default font size.
-			return pos.row * 18;
+			return this.viewer.session.getScreenLength() * this.viewer.renderer.lineHeight;
```

Comment:
Doesn't depend on `row` anymore.
Maybe `pow.row * this.viewer.renderer.lineHeight`?